alembic/README

```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```

Comment:
I think it be great to add some information to this document that outlines the normal procedure for making database schema changes.  Update models.py, run the autogenerate command, run the upgrade command, ...